cmake/emu/qemu.cmake

```
@@ -244,7 +244,7 @@ if(CONFIG_X86_64)
     ${CMAKE_OBJCOPY}
     -O elf32-i386
     $<TARGET_FILE:${logical_target_for_zephyr_elf}>
-    ${CMAKE_BINARY_DIR}/zephyr-qemu.elf
+    ${CMAKE_BINARY_DIR}/zephyr/zephyr-qemu.elf
```

We have `ZEPHYR_BINARY_DIR` for this purpose.
```suggestion
    ${ZEPHYR_BINARY_DIR}/zephyr-qemu.elf
```